code/WorkInProgress/pomf/spacepods/spacepods.dm

```diff
@@ -216,15 +216,15 @@
 		if("pod_civ")
 			desc = "A sleek civilian space pod."
 		if("pod_black")
-			desc = "An all black space pod with no insignias."
+			desc = "A space pod painted black."
 		if("pod_mil")
-			desc = "A dark grey space pod brandishing the Nanotrasen Military insignia"
+			desc = "A dark grey space pod bearing the Nanotrasen military insignia."
 		if("pod_synd")
-			desc = "A menacing military space pod with Fuck NT stenciled onto the side"
+			desc = "A menacing military space pod with \"Fuck NT\" stenciled onto the side."
 		if("pod_gold")
-			desc = "A civilian space pod with a gold body, must have cost somebody a pretty penny"
```

Comment:
I'd personally say keep the original wording and just fix the comma splice and missing period.